test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -141,4 +156,142 @@ public void AddAndRemoveNodeIT() throws Exception {
         shutdownCorfuServer(server_2);
         shutdownCorfuServer(server_3);
     }
+
+    @Test
+    public void clusterResizingTest1() throws Exception {
+        // This tests will resize the cluster according to the following
+        // order: add node -> add node -> remove node -> add node -> add node
```

Comment:
i think this comment is out of date. Can you make it a test level comment ?

Reply:
Done.